src/components/v2/ProgressBar/index.tsx

```diff
@@ -82,7 +80,7 @@ export const ProgressBarHorizontal = ({
       min={min}
       max={max}
       size="medium"
-      disabled={isDisabled}
+      disabled
```

Comment:
Should this be statically disabled?

Reply:
yes. we dont't have any cases with changing values of slider manually by user